src/SignalR/samples/ClientSample/UploadSample.cs

```diff
@@ -46,44 +48,104 @@ public static async Task BasicInvoke(HubConnection connection)
             foreach (var c in "hello")
             {
                 await channel.Writer.WriteAsync(c.ToString());
+                await Task.Delay(1000);
             }
             channel.Writer.TryComplete();
 
             var result = await invokeTask;
             Debug.WriteLine($"You message was: {result}");
         }
 
-        private static async Task WriteStreamAsync<T>(IEnumerable<T> sequence, ChannelWriter<T> writer)
+        public static async Task ScoreTrackerExample(HubConnection connection)
         {
-            foreach (T element in sequence)
+            // Andrew please add the updated code from your laptop here
+
+            var channel_one = Channel.CreateBounded<int>(2);
+            var channel_two = Channel.CreateBounded<int>(2);
+            _ = WriteItemsAsync(channel_one.Writer, new[] { 2, 2, 3 });
+            _ = WriteItemsAsync(channel_two.Writer, new[] { -2, 5, 3 });
+
+            var result = await connection.InvokeAsync<string>("ScoreTracker", channel_one.Reader, channel_two.Reader);
```

Comment:
Have we considered supporting ChannelWriter hub parameters in oder to support multiple response streams per invocation? It stand to reason that ChannelWriter parameters should work if ChannelReader.

Reply:
Yea it’s not a bad idea but not something to flesh out as part of this PR. Dylan investigatesd this and it seemed feasible.  Or something I would block on.